lib/koans/20_comprehensions.ex

```diff
@@ -0,0 +1,36 @@
+defmodule Comprehensions do
+  use Koans
+
+  @intro "Comprehensions"
+
+  # A comprehension is made of three parts: generators, filters and collectables.

```

Comment:
This would be a lovely `@intro`. Maybe even expanded to something like
```
@intro "A comprehension is made of three parts: generators, filters, and collectibles. We will look at how these interact with eachother".
```
I know other koans don't have a fancy `@intro`... its because I put the plumbing in place but never circled back to make them fun and read like a story. These koan could be refreshing comeback to that idea :smile: